internal/namespaces/account/v2alpha1/custom.go

```diff
@@ -39,18 +39,23 @@ func initCommand() *core.Command {
 
 func InitRun(ctx context.Context, argsI interface{}) (i interface{}, e error) {
 	// Get default local SSH key
-	relativePath := path.Join(".ssh", "id_rsa.pub")
-	filename := path.Join(core.ExtractUserHomeDir(ctx), relativePath)
-	shortenedFilename := "~/" + relativePath
-	localSSHKeyContent, err := ioutil.ReadFile(filename)
-
+	var shortenedFilename string
+	var err error
+	var localSSHKeyContent []byte
+	for _, keyName := range [3]string{"id_dsa.pub", "id_ed25519.pub", "id_rsa.pub"}  {
```

Comment:
Do we really want to use DSA? :scream:

Reply:
Another approach could be listing `*.pub` is the `.ssh` directory, and asking the user to chose one?

Reply:
Good point.
ssh supports the following formats: rsa, dsa, ecdsa and ed25519.

Which one do we want to support ?

Reply:
IMHO DSA should not be supported since it's been proven insecure, and is deprecated.

Also, I personally do not rely on `id_xxx` and prefer the name my keys, but we could just imply that a lot of users have 1 basic key :thinking: so this solution is still okay for me :smile:

Reply:
I replaced dsa with ecdsa https://github.com/scaleway/scaleway-cli/pull/1453/commits/acd4db90f4b519dce64091baf2097c1598f97042